packages/babel-core/src/transformation/util/clone-deep.ts

```diff
@@ -1,9 +1,32 @@
-import v8 from "v8";
-import cloneDeep from "./clone-deep-browser";
-
-export default function (value) {
-  if (v8.deserialize && v8.serialize) {
-    return v8.deserialize(v8.serialize(value));
+//https://github.com/babel/babel/pull/14583#discussion_r882828856
+function deepClone(value: any, cache: Map<any, any>): any {
```

Comment:
```suggestion
function deepClone<T extends object>(value: T, cache: Map<T, T>): T
```

Reply:
The problem is that `T` must be the union of all the types of the nested objects of `value` (since the `cache` Map potentially contains all of them), so "this function returns `T`" isn't much useful.

Reply:
Yes, this is an internal implementation that is not exported.